README.md

```diff
@@ -1,97 +1,330 @@
+Based on what we learned from analyzing the existing definitions of the top 20 terms, we concluded that improved plain language definitions are urgently needed for the UI lexicon in order to help claimants understand and navigate the system. " - [DOL's UX Lexicon research findings](https://www.dol.gov/agencies/eta/ui-modernization/use-plain-language/ui-lexicon#:~:text=Creating%20common%2Dsense%20plain%20language%20definitions%20for%20the%20UI%20lexicon%C2%A0)
+
+* "Due to the complexity of program rules and eligibility requirements, claimants often have difficulty understanding application questions, which can cause them to inadvertently provide incorrect answers and contributes to improper payments and spikes in call center call volume." - [DOL CX principles for online applications](https://www.dol.gov/agencies/eta/ui-modernization/customer-experience/improve-applications/cx-principles#:~:text=and%20occupation%20sections-,Use%20plain%20language,-Plain%20language%20is)
+
+## Recommended Practices
 
-## Recommended Practices:
 ### Have a dedicated “Income” section in your application
-<div class="section">
-  <img class="example" align="right" src="./images/dedicatedIncome.png"
-    alt="Identify who in the household has earned income, with examples." />
-  Collect income information in one dedicated section. If an application asks all income questions at the same time, the applicant can focus on: 
-  <ul>
-    <li>Gathering any needed information or documents (like paystubs).</li>
-    <li>Entering all income information during one session.</li>
-  </ul>
-  This helps clients avoid context-switching and encourages them to complete the income section.
+
+<div class="row">
+  <div class="column">
+Our research shows that clients do not separate the money their household receives as different types of income. Some agencies, however, have different sections for earned and unearned income in their online applications. Instead, collect income information from all members of the household in one dedicated section. If an application asks all income questions at the same time, the client can focus on:
+
+<ul>
+<li>Gathering any needed information or documents, like paystubs, to represent everyone in the household's income situation.</li>
+<li>Entering all income information during one session, including income that is not paid by an employer.</li>
+</ul>
+
+<a href="[url](https://www.nngroup.com/articles/minimize-cognitive-load/)">Research done on cognitive load</a> (the mental resources needed for a user to operate a system and complete tasks) shows that usability best practices, such as chunking alike content, helps reduce extraneous cognitive load. Reducing extraneous cognitive load means clients can complete applications faster and with higher accuracy. A simple approach like consolidating income questions into the same section can help clients avoid context-switching and encourages them to complete the income section.
```

Comment:
Should we think of a better phrase than "chunking alike content"? It made my body tense up. Maybe:
"placing similar content together"